strings/src/main/java/net/skillfactory/StringManagement.java

```diff
@@ -0,0 +1,139 @@
+package net.skillfactory;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.regex.Pattern;
+
+public class StringManagement {
+
+    public StringManagement(){
+    }
+
+    public boolean palindrome(String word){
+
+        String wordWithoutSpaces = word.replace(" ", "");
+
+        StringBuilder stringBuilder = new StringBuilder(wordWithoutSpaces);
+        stringBuilder = stringBuilder.reverse();
+
+        if (wordWithoutSpaces.equalsIgnoreCase((stringBuilder.toString()))){
+            return true;
+        } else {
+            return false;
+        }
```

Comment:
```suggestion
        return wordWithoutSpaces.equalsIgnoreCase((stringBuilder.toString())){            
```
Esto hace exactamente lo mismo